spec.bs

```diff
@@ -376,22 +376,23 @@ To enroll a user for Secure Payment Confirmation, relying parties should call
 {{AuthenticationExtensionsClientInputs/payment}} [=WebAuthn Extension=]
 specified.
 
-Note: We currently require an extension in order to allow credential creation
-      in an iframe and to allow the browser to cache SPC credentials in the
-      absence of [=WebAuthn Conditional UI=]. Future versions of this
-      specification may remove the requirement for the extension.
+Note: We currently require an extension in order to allow cross-origin
+      credential creation in an iframe and to allow the browser to cache
+      SPC credentials in the absence of [=WebAuthn Conditional UI=].
+      Future versions of this specification may remove the requirement
+      for the extension.
 
 # Authentication # {#sctn-authentication}
 
 To authenticate a payment via Secure Payment Confirmation, this specification
 defines a new [=payment method=], "[=secure-payment-confirmation=]". This
 payment method confirms the transaction with the user and then performs an
 [=authentication ceremony=] to authenticate the user and create a signed blob
-representing the transaction.
+representing the payment-specific information.
```

Comment:
Is this misleading? The signed blob is over the whole WebAuthn ceremony, not just the payment-specific bits. But *shrug*, don't particularly care here :D

Reply:
I'll change it to the authentication ceremony.